cmd/website/macros/macros.go

```diff
@@ -66,7 +66,21 @@ func expandCf(line string) string {
 		} else {
 			sep = ", "
 		}
-		fmt.Fprintf(&buf, "%s[`%s`](#%s)", sep, target, target)
+		if i := strings.IndexRune(target, ':'); i == -1 {
```

Comment:
It will be much clearer to abstract these into a function:

```go
func makeLink(cf string) string
```

Reply:
Agreed.